tests/ci/spec_to_test_mapping.py

```diff
@@ -118,11 +118,6 @@
         '',
         ''
     ],
-    'components/parallel-libs/trilinos/SPECS/trilinos.spec': [
```

Comment:
Isn't this mapping still useful for local runs ?

Reply:
Good point!